telemetry/definitions/commonDefinitions.json

```diff
@@ -3145,6 +3190,80 @@
                 { "type": "codecatalyst_devEnvironmentWorkflowStep"},
                 { "type": "codecatalyst_devEnvironmentWorkflowError", "required": false }
             ]
+        },
+        {
+            "name": "codewhispererterminal_userLoggedIn",
+            "description": "Emitted when users log in",
+            "passive": false,
+            "metadata": []
+        },
+        {
+            "name": "codewhispererterminal_completionInserted",
+            "description": "Emitted on Autocomplete user decisions",
+            "passive": false,
+            "metadata": [
+                { "type": "codewhispererterminal_command" },
+                { "type": "codewhispererterminal_duration" }
+            ]
+        },
+        {
+            "name": "codewhispererterminal_ghostTextActioned",
+            "description": "Emitted on GhostText user decisions",
+            "passive": false,
+            "metadata": [
+                { "type": "codewhispererterminal_duration" },
+                { "type": "codewhispererterminal_accepted" },
+                { "type": "codewhispererterminal_typedCount" },
+                { "type": "codewhispererterminal_suggestedCount" }
+            ]
+        },
+        {
+            "name": "codewhispererterminal_translationActioned",
+            "description": "Emitted on Translation user decisions",
+            "passive": false,
+            "metadata": [
+                { "type": "codewhispererterminal_duration" },
+                { "type": "codewhispererterminal_timeToSuggestion" },
+                { "type": "codewhispererterminal_accepted" }
+            ]
+        },
+        {
+            "name": "codewhispererterminal_cliSubcommandExecuted",
+            "description": "Emitted on CW CLI subcommand executed",
+            "passive": false,
+            "metadata": [
+                { "type": "codewhispererterminal_itemName" }
+            ]
+        },
+        {
+            "name": "codewhispererterminal_doctorExecuted",
+            "description": "Emitted when CW CLI user fails a doctor check",
+            "passive": false,
+            "metadata": [
+                { "type": "codewhispererterminal_itemName" }
+            ]
+        },
+        {
+            "name": "codewhispererterminal_dashboardPageViewed",
+            "description": "Emitted when CW CLI user fails a doctor check",
+            "passive": false,
+            "metadata": [
+                { "type": "codewhispererterminal_route" }
+            ]
+        },
+        {
+            "name": "codewhispererterminal_menuBarClicked",
+            "description": "Emitted when a user clicks the menu bar",
```

Comment:
you could have a click count as discussed in slack. collapsing it to have number of counts for a duration or session